app/javascript/alchemy_admin/utils/ajax.js

```diff
@@ -53,6 +53,7 @@ export default function ajax(method, path, data, accept = "application/json") {
   xhr.open(method, url.toString())
   xhr.setRequestHeader("Content-type", "application/json; charset=utf-8")
   xhr.setRequestHeader("Accept", accept)
+  xhr.setRequestHeader("X-Requested-With", "XMLHttpRequest")
```

Comment:
Maybe we should add a comment why this is necessary "For Rails' request.xhr?"

Also a test should verify the header is set.